mne/connectivity/spectral/envelope.py

```diff
@@ -0,0 +1,68 @@
+# Authors: Denis A. Engemann <denis.engemann@gmail.com>
+#          Sheraz Khan <sheraz@nmr.mgh.harvard.edu>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+
+
+def compute_corr(x, y):
+    """Correlate 2 matrices along last axis.
+
+    Parameters
+    ----------
+    x : np.ndarray of shape(n_time_series, n_times)
+        The first set of vectors.
+    y : np.ndarray of shape(n_time_series, n_times)
+        The second set of vectors.
+
+    Retrurns
```

Comment:
dengmannisms :-)

Reply:
Shouldn't this one be more generally available? Under mne.utils maybe, right by mne.utils.compute_corr?

Reply:
Ja we can deal with it later. Now we have other obstacles in moving this forward.

Reply:
I.e. the retrurns stay :D